homeassistant/components/point/__init__.py

```diff
@@ -0,0 +1,304 @@
+"""
+Support for Minut Point.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/point/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_TOKEN, CONF_WEBHOOK_ID
+from homeassistant.core import callback
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.dispatcher import (
+    async_dispatcher_connect, async_dispatcher_send)
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.util.dt import as_local, parse_datetime, utc_from_timestamp
+
+from . import config_flow  # noqa  pylint_disable=unused-import
+from .const import (
+    CLIENT_ID, CLIENT_SECRET, CONF_WEBHOOK_URL, DOMAIN, EVENT_RECEIVED,
+    NEW_DEVICE, SCAN_INTERVAL, SIGNAL_UPDATE_ENTITY, SIGNAL_WEBHOOK)
+
+REQUIREMENTS = ['pypoint==1.0.5']
+DEPENDENCIES = ['webhook']
+
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN:
+        vol.Schema({
+            vol.Required(CLIENT_ID): cv.string,
```

Comment:
All config keys constants should start with `CONF_`.

Reply:
Where is the coding standard that specify this or is this your personal preference?

Reply:
Standards are good but it is hard to follow them if you don't know about them.